app/frontend/components/domains/requirements-library/requirement-field-edit/editable-group.tsx

```diff
@@ -1,55 +1,76 @@
 import { useTranslation } from "react-i18next"
-import { EEnergyStepCodeDependencyRequirementCode } from "../../../../types/enums"
+import { EEnergyStepCodeDependencyRequirementCode, ERequirementType } from "../../../../types/enums"
 import { isStepCodePackageFileRequirementCode } from "../../../../utils/utility-functions"
+import { EditBox } from "./edit-box"
 import { EditableHelperText, TEditableHelperTextProps } from "./editable-helper-text"
+import { EditableInstructionsText, TEditableInstructionsTextProps } from "./editable-instructions-text"
 import { EditableLabel, TEditableLabelProps } from "./editable-label"
 import { IsElectiveCheckbox } from "./is-elective-checkbox"
 import { IsMultipleFilesCheckbox } from "./is-multiple-files-checkbox"
 import { IsOptionalCheckbox } from "./is-optional-checkbox"
 import { TIsElectiveCheckboxProps, TIsMultipleFilesCheckboxProps, TIsOptionalCheckboxProps } from "./types"
 
 export type TEditableGroupProps<TFieldValues extends FieldValues> = {
-  editableLabelProps?: TEditableLabelProps<TFieldValues>
+  requirementCode?: ERequirementType
+  editableLabelProps: TEditableLabelProps<TFieldValues>
   editableHelperTextProps?: TEditableHelperTextProps<TFieldValues>
   isOptionalCheckboxProps: TIsOptionalCheckboxProps<TFieldValues>
   isElectiveCheckboxProps: TIsElectiveCheckboxProps<TFieldValues>
   isMultipleFilesCheckboxProps?: TIsMultipleFilesCheckboxProps<TFieldValues>
-  editableInput?: JSX.Element
   multiOptionEditableInput?: JSX.Element
-  requirementCode: string
+  label: string
+  helperText?: string
+  editableInput?: ReactNode
+  editableInstructionsTextProps?: TEditableInstructionsTextProps<TFieldValues>
 } & Partial<StackProps>
 
 export function EditableGroup<TFieldValues>({
+  requirementCode,
   editableLabelProps,
   editableHelperTextProps,
-  editableInput,
-  multiOptionEditableInput,
   isOptionalCheckboxProps,
   isElectiveCheckboxProps,
   isMultipleFilesCheckboxProps,
-  requirementCode,
+  multiOptionEditableInput,
+  label,
+  helperText,
+  editableInput,
+  editableInstructionsTextProps,
   ...containerProps
 }: TEditableGroupProps<TFieldValues>) {
   const { t } = useTranslation()
+  const { isOpen, onOpen, onClose } = useDisclosure()
   const isEditLimited =
     isStepCodePackageFileRequirementCode(requirementCode) ||
     Object.values(EEnergyStepCodeDependencyRequirementCode).includes(
       requirementCode as EEnergyStepCodeDependencyRequirementCode
     )
+
   return (
     <Stack spacing={4} {...containerProps}>
       <EditableLabel {...editableLabelProps} />
-      {editableInput}
-      {editableInput && <EditableHelperText {...editableHelperTextProps} />}
-      {multiOptionEditableInput && (
-        <Stack>
-          {multiOptionEditableInput}
-          <EditableHelperText {...editableHelperTextProps} />
-        </Stack>
-      )}
+      <Flex flex={1}>
+        {editableInput ? (
+          <EditBox label={label} helperText={helperText}>
```

Comment:
can you explain to me what you're trying to accomplish here?

Reply:
<img width="805" height="336" alt="image" src="https://github.com/user-attachments/assets/9afba20d-e8e5-449f-8590-9319ba507eed" />

The UI now looks like this

Reply:
(I temporarily removed the "not found" fallback)